google/cloud/pubsub/quickstart/quickstart.cc

```diff
@@ -27,18 +27,19 @@ int main(int argc, char* argv[]) try {
 
   // Create a namespace alias to make the code easier to read.
   namespace pubsub = ::google::cloud::pubsub;
+
   auto publisher = pubsub::Publisher(
       pubsub::MakePublisherConnection(pubsub::Topic(project_id, topic_id)));
   auto id =
       publisher
           .Publish(pubsub::MessageBuilder{}.SetData("Hello World!").Build())
           .get();
   if (!id) throw std::move(id).status();
-  std::cout << "Hello World published with id=" << *id << "\n";
+  std::cout << "Hello World published with id=" << *id << std::endl;
```

Comment:
`std::endl` both prints a newline and flushes the output.  You almost certainly do not need to flush the output in these examples?

Reply:
Removed the endls across the quickstart